### PR TITLE
Create and show user created

### PR DESCRIPTION
**Pull request by thiennv57**

### CODE REVIEW

**Comment by VuongLong** [comment]
ok

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class UsersController < ApplicationController
+  before_action :init_user, only: [:show]
   def show
   end
```
Comment (ThuBM) [inline app/controllers/users_controller.rb]: indent

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class UsersController < ApplicationController
+  before_action :init_user, only: [:show]
```
Comment (ThuBM) [inline app/controllers/users_controller.rb]: add new line after this line

Comment (thiennv57) [inline app/controllers/users_controller.rb]: i've add a line after that

**Comment by ThuBM** [comment]
OK

**Pull request merged**